src/components/Logbar/Logbar.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { BsPersonCircle } from "react-icons/bs";
+import { BsPencilSquare } from "react-icons/bs";
+import { BsLockFill } from "react-icons/bs";
+
+import "./Logbar.scss";
+
+const Logbar = () => {
+  const [selectedOption, setSelectedOption] = useState("login");
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
```

Comment:
Consider adding here useNavigate to change the page depending on the selected option

Reply:
useLocation works here, as you showed